tiled/server/core.py

```diff
@@ -343,7 +348,7 @@ def construct_array_response(
 
 
 def construct_dataframe_response(
-    serialization_registry, df, metadata, request_headers, format=None
+    serialization_registry, df, metadata, request_headers, format=None, specs=[]
```

Comment:
Avoid mutable defaults. (See for example: https://docs.python-guide.org/writing/gotchas/)